test/configuration/karma-common.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
i pulled common configuration from karma-local and karma-sauce into a separate file